docs/docs/default-actions.mdx

```diff
@@ -107,6 +107,32 @@ class ActionSessionStart(Action):
         return events
 ```
 
+If you want to access the metadata which was sent with the user message which triggered
+the session restart, you can access the special slot `session_started_metadata`:
```

Comment:
```suggestion
the session start, you can access the special slot `session_started_metadata`:
```

?

Reply:
yes, was wondering about it during the. self-review as well